rustfmt.toml

```diff
@@ -1 +1,5 @@
 edition = "2021"
+
+imports_granularity="Module"
```

Comment:
Please add the link to the doc in comments

Reply:
Added